src/main/java/frc/robot/Robot.java

```diff
@@ -44,7 +45,13 @@ public class Robot extends TimedRobot {
     XboxController driverController = new XboxController(Constants.DRIVER_CONTROLLER_ID);
     XboxController coDriverController = new XboxController(1);
     static AHRS navX = new AHRS(SPI.Port.kMXP);
-    AutonActionRunner auton;
+    SequentialActionRunner auton;
+    static ParallelActionRunner teleopActionRunner = new ParallelActionRunner();
+
+    public static ParallelActionRunner getTeleopActionRunner() {
+        return teleopActionRunner;
+    }
+
```

Comment:
teleopActionRunner is the ActionRunner we use in teleop, we make a getter for it so we can give it actions to run in inputtedCoDriverControlls.